github/github.go

```diff
@@ -84,6 +84,9 @@ const (
 
 	// https://developer.github.com/changes/2016-07-06-github-pages-preiew-api/
 	mediaTypePagesPreview = "application/vnd.github.mister-fantastic-preview+json"
+
+	// https://developer.github.com/v3/repos/traffic/
+	mediaTypeTrafficPreview = "application/vnd.github.spiderman-preview"
```

Comment:
There should be a `+json` suffix, see similar lines above.

For more context, see https://github.com/google/go-github/pull/337#discussion_r66712727.